app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str):
+    file_name_in, file_name_out = command.split()[1:]
+
+    if file_name_in and file_name_out and file_name_in != file_name_out:
```

Comment:
```suggestion
    if file_name_in != file_name_out:
```
also, you should check the input command